tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "resolveJsonModule": true,
+    "lib": [
+      "es2016",
```

Comment:
es2018!

Reply:
this is generated, so we would update in the TypeScript micro generator I believe.